src/components/AudioPlayer/QuranReaderHighlightDispatcher.tsx

```diff
@@ -146,4 +152,20 @@ const getHighlightedLocation = (
   return highlightedLocation;
 };
 
+/**
+ * check if currentTime is within range timestampFrom and timestampTo
+ *
+ * example:
+ * - timestampFrom = 10, timestampTo = 20, currentTime = 10 should return true
+ * - timestampFrom = 10, timestampTo = 20, currentTime = 11 should return true
+ * - timestampFrom = 10, timestampTo = 20, currentTime = 20 should return false
+ *
+ * @param {number} currentTime
+ * @param {number} timestampFrom
+ * @param {number} timestampTo
+ * @returns {boolean} isWithinRange
+ */
+const isCurrentTimeInRange = (currentTime: number, timestampFrom: number, timestampTo: number) =>
+  currentTime >= timestampFrom && currentTime < timestampTo;
+
```

Comment:
IMO, this is not a big enough code to be a function, it's probably more readable inline. 

<img width="1381" alt="2021-09-23 at 08 47 49@2x" src="https://user-images.githubusercontent.com/11590314/134465028-42070132-dad8-4682-af98-b63ff8d66b9a.png">

Not a strong opinion though, I'll leave the decision to you :)

Reply:
yeah, this function is actually very simple.

But since this is a function, we add docs and give examples. Which helps to make the coder is easier to understand IMO.

If we duplicate the docs in 4 places, it's a bit less clean.

So I will keep this function for now